chart/compass/charts/metris/values.yaml

```diff
@@ -38,12 +38,12 @@ provider:
 
 edp:
   datastream:
-    name: "consumption-metrics"
-    namespace: "kyma-dev"
+    name: "TBD"
+    namespace: "TBD"
     version: 1
-    env: "dev"
-  token: ""
-  url: "https://input.yevents.io"
+    env: "TBD"
+  token: "TBD"
+  url: "TBD"
```

Comment:
I think it should be reverted. Correct me if I'm wrong, @maksd

Reply:
@pkosiec it's OK, we can leave it like that.